src/components/SongList/index.js

```diff
@@ -0,0 +1,39 @@
+import {
+  mapObjIndexed,
+  prop,
+  reverse,
+  sortBy,
+} from 'ramda'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { firebaseConnect } from 'react-redux-firebase'
+import SongList from './SongList';
+
+const sortDescByVotes = compose(
+  reverse,
+  sortBy(prop('votes')),
+  Object.values,
+  mapObjIndexed((song, id) => ({ ...song, id }))
```

Comment:
this is fantastic... I really need to dive deeper into Ramda, this is such a nice succinct way of writing this.